Formula/r/rip2.rb

```diff
@@ -0,0 +1,38 @@
+class Rip2 < Formula
+  desc "Safe and ergonomic alternative to rm"
+  homepage "https://github.com/MilesCranmer/rip2"
+  url "https://github.com/MilesCranmer/rip2/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "e8519e21877c8883f9f2a700036c53bce62b5ee0afaef47a12780999457e2633"
+  license "GPL-3.0-or-later"
+  head "https://github.com/MilesCranmer/rip2.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: ".")
+
+    # Generate shell completions
+    (buildpath/"rip.bash").write Utils.safe_popen_read(bin/"rip", "completions", "bash")
+    (buildpath/"rip.fish").write Utils.safe_popen_read(bin/"rip", "completions", "fish")
+    (buildpath/"_rip").write Utils.safe_popen_read(bin/"rip", "completions", "zsh")
```

Comment:
you can use `generate_completions_from_executable(bin/"rip", "completion")` for these three

Reply:
okay will adjust, thanks.